src/Components/ReviewCard.jsx

```diff
@@ -1,21 +1,31 @@
-export const ReviewCard = ({ reviews }) => {
-  return (
+import { Link } from "react-router-dom";
+export const ReviewCard = ({ reviews, isLoading }) => {
+  return !isLoading ? (
     <section>
       <ul className="reviews">
         {reviews.map((review) => {
           return (
             <li key={review.review_id} className="review-card">
-              <h3>{review.title}</h3>
-              <img
-                src={review.review_img_url}
-                alt={review.title}
-                className="review-img"
-              />
+              <section>
+                <Link
+                  to={`/reviews/${review.review_id}`}
+                  id="review-card__link"
```

Comment:
Similar issue here to previous comment with the id not being unique to each tag.